x-pack/plugins/security_solution/public/common/lib/apm/user_actions.ts

```diff
@@ -33,3 +33,21 @@ export const RULES_TABLE_ACTIONS = {
   PREVIEW_ON: `${APP_UI_ID} rulesTable technicalPreview on`,
   PREVIEW_OFF: `${APP_UI_ID} rulesTable technicalPreview off`,
 };
+
+export const TIMELINE_ACTIONS = {
+  SAVE: `${APP_UI_ID} timeline save`,
+  DUPLICATE: `${APP_UI_ID} timeline duplicate`, // it includes duplicate template, create template from timeline and create timeline from template
+  DELETE: `${APP_UI_ID} timeline delete`,
+  BULK_DELETE: `${APP_UI_ID} timeline bulkDelete`,
+};
+
+export const ALERTS_ACTIONS = {
```

Comment:
👌🏾, thank you! Do we want to add the cases actions (add to new case, add to existing case) in the future, or have the cases team manage it?

Reply:
Yes, the Cases actions were implemented in this PR initially, but after talking with the Cases team, they seem to have very little bandwidth to review the changes for 8.5 🤷 
So I decided to extract the Cases actions in [a separate PR](https://github.com/elastic/kibana/pull/139937), this way we'll have at least the rest of the actions for 8.5.